misc/openapi-ballerina/modules/openapi-to-ballerina-generator/src/main/java/org/ballerinalang/openapi/typemodel/BallerinaIOpennApiRequestBody.java

```diff
@@ -0,0 +1,25 @@
+package org.ballerinalang.openapi.typemodel;
```

Comment:
class name is BallerinaIOpenApiRequestBody.java? What is "I" between "Ballerina" and "OpenApi" stands for?

Reply:
And there is an additional "n" in the "OpenApi"

Reply:
FIxed with commit cb03db8803b33e48835ec75daaac998460e16a22